### DISCUSSION TITLE
Custom header and footer based on variable or class.

### DISCUSSION THREAD

**Discussion by 2acode** [dompdf/dompdf#2332] [General]
Hello,
I use header and footer for my document.
Now I need to add an appendix to the created document that has no headers and footers.

I don't think there is a way to merge two documents (if so let me know) so I'm trying to set the header and footer for the appendix in the html document.
I cannot use the page number as both documents (order and appendix) have variable length.
The idea would be to use a variable in the script or a class but after several tests and having checked the other issues I have not found a solution.

So is there a way to set header and footer for each individual page?

Thanks

**Comment by bsweeney** [comment]
So on the appendix you want to remove the header/footer? Possibly doable but certainly not with straight HTML+CSS. Though Dompdf is unable to merge documents I think it would be useful to be able to render separate HTML+CSS in sequence into the same document. Unfortunately we do not have any plans at the moment to work on that functionality.

There are some good libraries for merging PDFs (like [libmergepdf](https://github.com/hanneskod/libmergepdf)). Have you looked into using any of those?

**Reply by 2acode** [reply]
Yes, i need to remove header/footer in appendix and yes, mergin i think is the solution (thanks for lib).
At the moment i'm trying also a solution using wkhtmltopdf adding pages.
Thanks
